opm/input/eclipse/Deck/DeckKeyword.cpp

```diff
@@ -291,6 +291,9 @@ namespace Opm {
         return this->getDataRecord().getDataItem().data_size();
     }
 
+    std::vector<int>& DeckKeyword::getIntData(){
```

Comment:
brace

Reply:
> brace

Right. I've pushed an update to address this.